src/frontend/src/lib/styles/theme/base-colors.scss

```diff
@@ -0,0 +1,7 @@
+$alice-blue: #e8f1ff;
+$water: #d1e3ff;
+$pale-cornflower-blue: #b0cdff;
+$french-sky-blue: #84b2ff;
+$crayola: #004abe;
```

Comment:
I think this is quite generic for a color name, we were calling it `cobalt`, no?

Reply:
Yeah I can change it, i used 	// https://www.color-name.com which is mentioned in colors.scss - I have no idea why it gave me different names xD